### ISSUE TITLE
GUI : FED : Move Event log API to Redfish 

### ISSUE THREAD

**Issue by gtmills** [ibm-openbmc/dev#2070]
For event logs, can use 
` https://${bmc}/redfish/v1/Systems/system/LogServices/EventLog/Entries`

instead of ` .get('/xyz/openbmc_project/logging/enumerate')`

https://github.com/openbmc/webui-vue/blob/97f4187e1bac021ea7b5a0f5a780a3d4004edf64/src/store/modules/Health/EventLogStore.js#L67

**Comment by gtmills** [comment]
This is blocked by not having the resolved property.. :(

**Comment by derick-montague** [comment]
I think we can close this as a duplicate of #1211

**Comment by yoshiemuranaka** [comment]
I wouldn't mind keeping it to help break down the event log page story

**Comment by derick-montague** [comment]
Great! I added a dependency and reassigned to you. Thanks, Yoshie!

**Comment by yoshiemuranaka** [comment]
Resolved by https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/32082
